### ISSUE TITLE
theta-crop.qtz ?

### ISSUE THREAD

**Issue by kosso** [kfarr/theta-s-quartz#2]
Where can I find theta-crop.qtz to try running this too?

thx.

**Comment by kfarr** [comment]
Hi Kosso, last night I combined both the theta-crop and theta-fish2sphere filters into one, now just called theta-fish2sphere. You can still access the old version here, click on "raw" to download:
https://github.com/kfarr/theta-s-quartz/blob/91b61a642f929df2fc76fea297450061e19878ee/theta-crop.qtz

I just updated the readme now to remove reference to the theta-crop filter

**Comment by kosso** [comment]
Cool. I just got Quartz Composer and now see that it's patched in ;)  Great work btw. 

I just tried it with the live camera feed via USB and it 'works' but obviously needs some sort of blending at the edges. 

I also note that the DualFishBlender places the 'front image in the middle and split/spans the 'back' camera on either side. 

I've never looked at Quartz before. It's really interesting to see what's possible.
I had a pretty low framerate on the output video though. Possibly due to me trying it on a MacBook Air.
